synfig-core/src/synfig/layers/layer_skeleton.cpp

```diff
@@ -167,14 +167,14 @@ Layer_Skeleton::sync_vfunc()
  		const Bone &bone = i->get(Bone());
  		Matrix matrix = bone.get_animated_matrix();
  		Vector v;
- 		cout<<i-list.begin()<<endl;
+ 		cout<<bone.get_name()<<endl;
+ 		cout<<"Angle:"<<acos(matrix.axis(0)[0])*180/(2*acos(0.0))<<endl;
```

Comment:
This code doesn't give correct value. For angle of bone = `-45` it shows `45`. And when angle of bone = `45` it displays `45` too.